mistralrs-core/src/pipeline/mod.rs

```diff
@@ -82,6 +83,37 @@ pub struct SimpleModelPaths<P> {
     lora_preload_adapter_info: Option<HashMap<String, (P, LoraConfig)>>,
 }
 
+impl<P> SimpleModelPaths<P> {
```

Comment:
This API was internal before, so not really well documented. Can you please add some docs to explain what each path should point to?

Reply:
Done

Reply:
That looks good. Thanks for adding it!